spec/jekyll_seo_tag_spec.rb

```diff
@@ -237,11 +237,49 @@
     end
 
     context "with absolute site.logo" do
-      let(:site) { make_site("logo" => "http://cdn.example.invalid/logo.png", "url" => "http://example.invalid") }
+      let(:site) { make_site("logo" => "http://cdn.example.invalid/logo.png", "url" => "http://example.invalid", "author" => "Mr. Foo") }
```

Comment:
I don't think I understand the connection between `author` and `logo`.
If we're adding something new, we should add new tests, _not_ modify existing tests.